### ISSUE TITLE
pip install -U yotta : errors out in Win10

### ISSUE THREAD

**Issue by monojk** [ARMmbed/yotta#860]
When trying to install yotta on Win10 I get the errors at the end. Probable cause:   
**It requires requests[security] and cryptography>=2.1.4,<=2.3. At the same time requests requires pyOpenSSL which requires cryptography>=2.8**

The title line is from stackoverflow where I reported the problem first, thus the [duplicate], sorry for the misleading text. The comment in stackoverflow states that this must be fixed in yotta.

From the installation:

```
Collecting yotta
  Using cached yotta-0.20.0.tar.gz (130 kB)
...
Collecting cryptography<=2.3,>=2.1.4
  Using cached cryptography-2.3.tar.gz (449 kB)
...
Collecting pyOpenSSL>=0.14; extra == "security"
  Using cached pyOpenSSL-19.1.0-py2.py3-none-any.whl (53 kB)
...
ERROR: pyopenssl 19.1.0 has requirement cryptography>=2.8, but you'll have cryptography 2.3 which is incompatible.
Installing collected packages: cryptography, pyOpenSSL, requests, PyJWT, wrapt, deprecated, PyGithub, colorama, hgapi, MarkupSafe, Jinja2, pathlib, jsonschema, argcomplete, mbed-test-wrapper, pyyaml, pyelftools, milksnake, appdirs, cmsis-pack-manager, prettytable, sortedcontainers, intervaltree, intelhex, pywinusb, pyusb, pyOCD, argparse, xmltodict, project-generator-definitions, project-generator, valinor, jsonpointer, ntfsutils, yotta
    Running setup.py install for cryptography ... error
...
    build\temp.win32-3.8\Release\_openssl.c(546): fatal error C1083: Cannot open include file: 'openssl/opensslv.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\cl.exe' failed with exit status 2
```

**Comment by thegecko** [comment]
PRs Welcome :)

**Comment by 0xc0170** [comment]
I was able to reproduce the problem :/ @monojk Have you already found a fix - bumped the version and tested it?

**Comment by monojk** [comment]
No, I don't have a fix. I'm just a user or trying to use Yotta for compiling micropython.

**Comment by monojk** [comment]
Any progress on this?

**Comment by thegecko** [comment]
@rwalton-arm Does your fix close this issue?

**Comment by rwalton-arm** [comment]
Yeah, my fix should have resolved this. I tested Windows 10 python 3.7 and 2.7 and the installation completed without issue.

**Comment by rwalton-arm** [comment]
Fix is in v0.20.5. Please let us know if you have any more issues.

**Comment by monojk** [comment]
Successfully installed. Thanks
